lib/electric_slide/call_queue.rb

```diff
@@ -62,7 +62,7 @@ def available_agent_summary
     # @return {Agent}
     def checkout_agent
       agent = @strategy.checkout_agent
-      agent.presence = :busy
+      agent.presence = :busy if agent
```

Comment:
It's worth noting that the server where you were testing was running a 5-day-old deploy. It's possible the fix wasn't included during the last round of testing.